search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/AuthorizationFilterTransformer.java

```diff
@@ -40,6 +40,6 @@ public SearchQuery toSearchQuery(final AuthorizationFilter filter) {
 
   @Override
   public List<String> toIndices(final AuthorizationFilter filter) {
-    return List.of("identity-authorizations-8.7.0_alias");
+    return List.of("camunda-authorization-8.7.0_alias");
```

Comment:
Just for posterity: I don't like and also do not accept that we have this hardcoded. But there is a PR in progress that will change this here https://github.com/camunda/camunda/pull/25048